frontend/src/APIClients/mutations/TestMutations.ts

```diff
@@ -45,3 +45,12 @@ export const CREATE_NEW_ASSESSMENT = gql`
     }
   }
 `;
+
+export const UPLOAD_TEST_IMAGE = gql`
```

Comment:
this is my bad, but can we update line 41 to `CREATE_NEW_TEST` to be consistent with the other mutations? also `Create_Test` -> `CreateTest`. sorry .-.